components/homme/test_execs/CMakeLists.txt

```diff
@@ -161,20 +161,25 @@ ENDIF ()
 IF(${BUILD_HOMME_SWEQX})
   ADD_SUBDIRECTORY(swtcA)
   ADD_SUBDIRECTORY(swtcB)
+  ADD_SUBDIRECTORY(swtcC)
 ENDIF()
 
 
 # Add the test exec subdirs for the preqx executable
 IF(${BUILD_HOMME_PREQX})
   ADD_SUBDIRECTORY(baroC)
   ADD_SUBDIRECTORY(baroCam)
+  ADD_SUBDIRECTORY(preqx-nlev10-native)
```

Comment:
This is a minor comment but with these new execs time building homme is doubled. should these execs be wrapped in some IF_PLANAR_TESTS variable?

Reply:
So right now there are the planar regression tests, which only need a few of the execs; and the full planar test suite (similar to a DCMIP-type suite of tests) that needs all of them. Only the regression tests should be run to check BFB pull requests; the full suite is useful for analyzing non-BFB changes and generating results for comparison with other models.

I can rework how the executables are built for the planar test suite so we can comment out most of them, if the build time is an issue.

Reply:
thanks -- probably, let's just leave it as is for now.

Reply:
Sounds good! I will make sure and put these changes in V2